<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>301</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Входные данные</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Размер массива   --&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sizeOfArrayLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Тип массива</string>
     </property>
     <widget class="QRadioButton" name="intRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Целочисленный</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="doubleRadioButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Вещественный</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер элемента [] --&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="indexLineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>240</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sizeOfArrayPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Подтвердить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addNewValueToArrPushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>681</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Результаты</string>
    </property>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>321</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Исходный массив</string>
     </property>
     <widget class="QListWidget" name="mainArrayListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>321</width>
       <height>391</height>
      </rect>
     </property>
     <property name="title">
      <string>Результат запроса</string>
     </property>
     <widget class="QListWidget" name="mainArrayListWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>301</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>301</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Выберите запрос</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>281</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Спочатку всі елементи, які не перевищують середнє арифметичне елементів масиву, потім всі інші.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Максимальний по модулю елемент масиву</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сума елементів масиву, розташованих між першим і останнім позитивними елементами</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Перетворити масив таким чином, щоб елементи, які дорівнюють нулю, розташовувалися після всіх інших</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="queryRequestPushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>281</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Выполнить</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>1003</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1022</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
